### ISSUE TITLE
Clean up warnings in reconstruction tests

### ISSUE THREAD

**Issue by sdmccabe** [netsiphd/netrd#124]
There's a lot of warning spam because the reconstruction tests aren't parameterizing the threshold step. This should be a one-line fix for each of the reconstruction tests.

**Comment by sdmccabe** [comment]
A common warning that I can't place is the following:

```{python}
tests/test_reconstruction.py::test_graph_size
tests/test_reconstruction.py::test_oce
  /home/main/.pyenv/versions/miniconda3-latest/lib/python3.6/site-packages/networkx/classes/digraph.py:1180: DeprecationWarning: G.fresh_copy is deprecated. Use G.__class__ instead
    H = self.fresh_copy()
```
I haven't managed to pin this one down, so it'll stay, but noting its presence. I don't think it's a big deal.

**Comment by sdmccabe** [comment]
Resolved by #126. There are still some warnings, but they're specific to methods, not a product of the test suite (e.g., #127).
